---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
redirect_from:
  - /projects
---

{% include base_path %}

I am generally interested in developing machine learning methods and applying them in impactful ways. I also enjoy building online resources that provide programmatic, intuitive access to data. My previous work has focused primarily on molecular biology, particularly in the context of human health and disease, but I'm open (and excited!) to explore new avenues after I receive my PhD from Princeton in December of 2024.

Below, I'll highlight a few projects that have been important to me:

**[Innate Immune Memory in Influenza Infection](https://www.biorxiv.org/content/10.1101/2024.09.20.612974v1) (First Author, Preprint on bioRxiv)**

Viral infections can induce changes in innate immunity that persist after virus clearance. Here, we used blood samples from a human influenza H3N2 challenge study to perform comprehensive multi-omic analyses. We detected remodeling of immune programs in innate immune cells after resolution of the infection that was proportional in magnitude to the level of prior viral load. We found changes associated with suppressed inflammation including decreased cytokine and AP-1 gene expression as well as decreased accessibility at AP-1 targets and interleukin-related gene promoter regions. We also found decreased histone deacetylase gene expression, increased MAP kinase gene expression, and increased accessibility at interferon-related gene promoter regions. Genes involved in inflammation and epigenetic-remodeling showed modulation of gene-chromatin site regulatory circuit activity. These results reveal a coordinated rewiring of the epigenetic landscape in innate immune cells induced by mild influenza virus infection.

To learn more, please visit the [associated manuscript](https://www.biorxiv.org/content/10.1101/2024.09.20.612974v1).

**[SPEEDI (Single-cell Pipeline for End to End Data Integration)](https://www.biorxiv.org/content/10.1101/2023.11.01.564815v2) (Co-first author, Preprint on bioRxiv, Accepted in Cell Systems)**

To facilitate single-cell multi-omics analysis and improve reproducibility, we present SPEEDI (Single-cell Pipeline for End to End Data Integration), a fully automated end-to-end framework for batch inference, data integration, and cell type labeling. SPEEDI introduces data-driven batch inference and transforms the often heterogeneous data matrices obtained from different samples into a uniformly annotated and integrated dataset. Without requiring user input, it automatically selects parameters and executes pre-processing, sample integration, and cell type mapping. It can also perform downstream analyses of differential signals between treatment conditions and gene functional modules. SPEEDI’s data-driven batch inference method works with widely used integration and cell-typing tools. By developing data-driven batch inference, providing full end-to-end automation, and eliminating parameter selection, SPEEDI improves reproducibility and lowers the barrier to obtaining biological insight from these valuable single-cell datasets.

To learn more, please visit the [SPEEDI R package](https://github.com/FunctionLab/SPEEDI), the [SPEEDI interactive web application](https://speedi.princeton.edu/), or the [SPEEDI manuscript](https://www.biorxiv.org/content/10.1101/2023.11.01.564815v2).

**Extracellular RNAs**

Before coming to graduate school, I worked as a software engineer and data analyst for [Dr. Aleksandar Milosavljevic](https://scholar.google.com/citations?hl=en&user=BjncDdoAAAAJ) in the [Bioinformatics Research Laboratory](http://genboree.org/site/bioinformatics_research_laboratory) at Baylor College of Medicine in Houston, TX. In that role, I contributed to several major research projects for the NIH Common Fund’s Extracellular RNA Communication (ERC) program. The purpose of the program was to study extracellular RNAs (exRNAs) across many different research areas, such as better understanding of exRNA biogenesis, discovery of exRNA biomarkers and use of exRNAs in therapies, and construction of a large set of reference profiles associated with healthy subjects as a foundation for future clinical research. As a leading member of the program’s Data Management and Resource Repository (DMRR), I presented at numerous conferences and helped create public resources to aid the wider scientific community in exRNA research.

For example, I played a key role in the design and development of the [exRNA Atlas](https://exrna-atlas.org), the primary online repository for extracellular RNA data generated by members of the ERC program. I designed and implemented a web-based [JSON-LD](https://json-ld.org/) API that incorporated [FAIR](https://www.go-fair.org/fair-principles/) criteria and offered programmatic access to data and associated metadata stored in the Atlas. I also created [thorough documentation](https://exrna-atlas.docs.stoplight.io/) on how to use the API. As of May 13, 2021, the exRNA Atlas contained data and metadata from 7,756 samples, with contributions stemming from 42 different studies and 21 different labs. As a co-first author, I helped write a highly collaborative paper ([published in Cell](https://www.ncbi.nlm.nih.gov/pubmed/30951672)) that both described the Atlas resource and provided an integrative data analysis, backed by wet lab experimental validation, showing strong evidence for the existence of distinct exRNA cargo types across human biofluids.

I also made significant contributions to the ERC program’s extra-cellular RNA processing toolkit ([exceRpt](http://github.gersteinlab.org/exceRpt/)), a small and long RNA-seq data processing pipeline specialized for human and mouse exRNA data. In collaboration with [Dr. Mark Gerstein’s lab](http://www.gersteinlab.org/) at Yale University, the creators of exceRpt, I implemented numerous bug fixes and enhancements to exceRpt, improving the robustness of the pipeline. I also made exceRpt more accessible to the wider scientific community by integrating it into the [Genboree Workbench](http://genboree.org/java-bin/workbench.jsp), a web-based platform that hosts a variety of different bioinformatics tools and stores related data for users. By submitting sample data for processing via the Workbench, users took advantage of a computing cluster and a highly parallel implementation of exceRpt, allowing for data from dozens of samples to be processed simultaneously. The Workbench version of exceRpt has been extensively used, with small RNA-seq data from over 26,000 samples processed since the tool was integrated in 2014. A paper describing our work on exceRpt was published in [Cell Systems](https://www.ncbi.nlm.nih.gov/pubmed/30956140).

**Making Predictions to Learn More About Human Health and Disease**

Nowadays, as a member of [Dr. Olga Troyanskaya’s lab](https://function.princeton.edu/), I'm using machine learning to make predictions in the context of human health and disease. More coming soon!
