---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
redirect_from:
  - /projects
---

{% include base_path %}

I am generally interested in developing machine learning methods and applying them in impactful ways. I also enjoy building online resources that provide programmatic, intuitive access to data. My previous work has focused primarily on molecular biology, particularly in the context of human health and disease, but I'm open (and excited!) to explore new avenues after I receive my PhD from Princeton in December of 2024.

Below, I'll highlight a few projects that have been important to me:

---

**[Innate Immune Memory in Influenza Infection](https://www.biorxiv.org/content/10.1101/2024.09.20.612974v1) (First Author, Preprint on bioRxiv)**

Viral infections can induce changes in innate immunity that persist after virus clearance. Here, we used blood samples from a human influenza H3N2 challenge study to perform comprehensive multi-omic analyses. We detected remodeling of immune programs in innate immune cells after resolution of the infection that was proportional in magnitude to the level of prior viral load. We found changes associated with suppressed inflammation including decreased cytokine and AP-1 gene expression as well as decreased accessibility at AP-1 targets and interleukin-related gene promoter regions. We also found decreased histone deacetylase gene expression, increased MAP kinase gene expression, and increased accessibility at interferon-related gene promoter regions. Genes involved in inflammation and epigenetic-remodeling showed modulation of gene-chromatin site regulatory circuit activity. These results reveal a coordinated rewiring of the epigenetic landscape in innate immune cells induced by mild influenza virus infection.

To learn more, please visit the [associated manuscript](https://www.biorxiv.org/content/10.1101/2024.09.20.612974v1).

---

**[SPEEDI (Single-cell Pipeline for End to End Data Integration)](https://www.biorxiv.org/content/10.1101/2023.11.01.564815v2) (Co-first author, Preprint on bioRxiv, Accepted in Cell Systems)**

To facilitate single-cell multi-omics analysis and improve reproducibility, we present SPEEDI (Single-cell Pipeline for End to End Data Integration), a fully automated end-to-end framework for batch inference, data integration, and cell type labeling. SPEEDI introduces data-driven batch inference and transforms the often heterogeneous data matrices obtained from different samples into a uniformly annotated and integrated dataset. Without requiring user input, it automatically selects parameters and executes pre-processing, sample integration, and cell type mapping. It can also perform downstream analyses of differential signals between treatment conditions and gene functional modules. SPEEDI’s data-driven batch inference method works with widely used integration and cell-typing tools. By developing data-driven batch inference, providing full end-to-end automation, and eliminating parameter selection, SPEEDI improves reproducibility and lowers the barrier to obtaining biological insight from these valuable single-cell datasets.

To learn more, please visit the [SPEEDI R package](https://github.com/FunctionLab/SPEEDI), the [SPEEDI interactive web application](https://speedi.princeton.edu/), or the [SPEEDI manuscript](https://www.biorxiv.org/content/10.1101/2023.11.01.564815v2).

---

**[Virtual Repository](https://genboree.org/vbr-hub/)**

I played a key role in the development of the Extracellular RNA Communication (ERC) program’s Virtual Biorepository (VBR) resource, a web-based service that mediates biological sample transfers between sample providers and requesters. As the project liaison for the VBR, I gave oral and poster presentations at numerous conferences, participated in monthly organizational calls with stakeholders, and provided significant feedback to programmers on our team for fixing bugs and improving features.

---

**[The Extracellular RNA Atlas](https://www.cell.com/cell/fulltext/S0092-8674(19)30167-9?sf210646493=1) (Co-first author, Published in Cell)**

I played a key role in the design and development of the exRNA Atlas, the primary online repository for extracellular RNA data generated by members of the Extracellular RNA Communication (ERC) program. I made important contributions to various Atlas features and tools, such as an ncRNA search feature to discover expression of selected ncRNAs across different biofluid data present in the Atlas, FTP data submission pipelines for RNA-seq and qPCR data with extensive support for linking associated metadata, and integrated downstream analysis tools for exRNA data (e.g., BioGPS for visualization of ncRNA expression, DESeq2 for pairwise differential expression of miRNAs across certain key metadata fields, and WikiPathways’ Pathway Finder for pathway analysis of differentially expressed miRNAs). I also designed and implemented a web-based JSON-LD API that incorporated various FAIR (Findability, Accessibility, Interoperability, and Reusability) criteria and offered programmatic access to data and associated metadata stored in the Atlas.

As of July 15, 2021, the Atlas contained data and metadata from 7,756 samples, with contributions stemming from 42 different studies and 21 different labs. Many studies were translational in nature, focusing on ncRNA biomarker discovery for specific conditions (e.g., multiple sclerosis, gastric cancer, and placental dysfunction). As a co-first author, I helped write a highly collaborative [paper](https://www.cell.com/cell/fulltext/S0092-8674(19)30167-9?sf210646493=1) (published in Cell) that both described the Atlas resource and provided an integrative data analysis, backed by wet lab experimental validation, showing strong evidence for the existence of distinct exRNA cargo types across human biofluids.

---

**[Extra-cellular RNA Processing Toolkit (exceRpt)](https://www.cell.com/cell/fulltext/S0092-8674(19)30167-9?sf210646493=1) (Published in Cell Systems)**

I made significant contributions to the Extracellular RNA Communication (ERC) program’s extra-cellular RNA processing toolkit (exceRpt), a small and long RNA-seq data processing pipeline specialized for human and mouse exRNA data. I made exceRpt more accessible to the wider scientific community by helping to integrate it into the Genboree Workbench, a web-based platform that hosts a variety of different bioinformatics tools and stores related data for users. By submitting sample data for processing via the Workbench, users took advantage of our computing cluster and a highly parallel implementation of exceRpt, allowing for data from dozens of samples to be processed simultaneously. The Workbench version of exceRpt has been extensively used, with small RNA-seq data from over 26,000 samples processed since the tool was integrated in 2014. A [paper](https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30074-2?sf210646498=1) describing our work on exceRpt was published in Cell Systems.
